azurelinuxagent/ga/exthandlers.py

```diff
@@ -979,39 +978,42 @@ def report_ext_handlers_status(self, incarnation_changed=False):
                       is_success=False,
                       message=msg)
 
-    @staticmethod
-    def write_ext_handlers_status_to_info_file(vm_status):
-        status_path = os.path.join(conf.get_lib_dir(), AGENT_STATUS_FILE)
-
-        agent_details = {
-            "agent_name": AGENT_NAME,
-            "current_version": str(CURRENT_VERSION),
-            "goal_state_version": str(GOAL_STATE_AGENT_VERSION),
```

Comment:
did we remove goal_state_version?

Reply:
Goal state version now comes from the status_blob_data 
  "aggregateStatus": {
    "guestAgentStatus": {
      "version": "9.9.9.9", // Goal state version
      "status": "Ready",
      "formattedMessage": {
        "lang": "en-US",
        "message": "Guest Agent is running"
      }
    }

Reply:
ok